testsuite/tests/conftest.py

```diff
@@ -930,7 +933,8 @@ def finalizer():
                             hook(svc)
                         except Exception:  # pylint: disable=broad-except
                             pass
-
+                    for usage in svc.backend_usages.list():
```

Comment:
This is reached after backend_usages are deleted, unless order of fixtures is changed in a tests to have `backend_usages` before `service`.